Storage/Storage/Tools/StorageType+Deletions.swift

```diff
@@ -118,4 +118,15 @@ public extension StorageType {
             deleteObject($0)
         }
     }
+
+    /// Deletes all of the stored `StalePlugin` entities with a specified `siteID` whose name is not included in `currentPluginNames` array.
+    ///
+    func deleteStalePlugins(siteID: Int64, currentPluginNames: [String]) {
```

Comment:
```suggestion
    func deleteStalePlugins(siteID: Int64, activePluginNames: [String]) {
```
I think `active` expressed the intention better than `current`. What do you think?

Reply:
I'm not quite happy with `current` either, but `active` does not seem right since plugins can be active/inactive, so the name can be ambiguous. How about `valid`? 🤷‍♀️

Reply:
I think we can go with `installed` plugins!